src/otn/linkeddatex2/Metadata.php

```diff
@@ -13,20 +13,52 @@
 {
     private static function addTriple(&$graph, $subject, $predicate, $object) {
         array_push($graph, [
-            'graph' => 'Metadata',
+            'graph' => '#Metadata',
             'subject' => $subject,
             'predicate' => $predicate,
             'object' => $object
         ]);
     }
 
+    public static function add_counts_to_multigraph(&$multigraph) {
+        $dotenv = new Dotenv\Dotenv(__DIR__ . "/../../../");
+        $dotenv->load();
+        $base_url = $_ENV["BASE_URL"];
+
+        $graph_counts = array();
+        $triples = 0;
+        foreach ($multigraph as $quad) {
+            if ($quad['graph'] !== "") {
+                if (!isset($graph_counts[$quad['graph']])) {
+                    $graph_counts[$quad['graph']] = 1;
+                } else {
+                    $graph_counts[$quad['graph']]++;
+                }
+            }
+            $triples++;
+        }
+        foreach ($graph_counts as $graph => $count) {
+            array_push($multigraph, [
+                'graph' => $graph,
+                'subject' => $graph,
+                'predicate' => 'void:triples',
+                'object' => $graph_counts[$graph] + 1
+            ]);
+            $triples++;
+        }
+        array_push($multigraph, [
+            'subject' => $base_url,
+            'predicate' => 'void:triples',
+            'object' => $triples + 1
```

Comment:
This should be a literal and not a resource. Create this with: 
`'object' => \pietercolpaert\hardf\Util::createLiteral($triples + 1, 'http://www.w3.org/2001/XMLSchema#integer')`